addons/gamification/models/gamification_badge.py

```diff
@@ -95,37 +95,21 @@ def _get_owners_info(self):
             'granted_users_count': 0,
             'unique_owner_ids': [],
         }
-        if not self.ids:
+        if not self:
```

Comment:
I think that the `.ids` was correct here as we don't want to do the query if the compute is done on new records?